addon/components/flash-message.js

```diff
@@ -59,6 +60,14 @@ export default Component.extend({
     }
   }),
 
+  // Observer used here for backwards compatibility with Ember 1.11/1.12, replace
+  // with on('didRender') once possible.
+  _startTimers: on('didInsertElement', observer('flash', function() {
```

Comment:
Why does this need to be an observer? It only needs to fire one time when inserted into the DOM.